website/static/docs/external_plugins/examples/sshfs.sh

```diff
@@ -260,18 +260,27 @@ if [[ "${path}" == "" ]]; then
     exit 0
   ;;
   "exec")
-    cmd="$3"
-
+    opts="$4"
+    cmd="$5"
+    shift
+    shift
     shift
     shift
     shift
     args="$@"
 
-    # exec'ing <cmd> <args> on a VM is equivalent to exec'ing it
-    # on the VM via. ssh (vm_exec)
+    # First we parse the provided Exec options. Wash guarantees that
+    # the passed-in options are valid JSON and that they are all present,
+    # so we don't need to do our own validation.
+    #
+    # NOTE: Only the "tty" option is relevant. We don't need to worry about
+    # "elevate" because we are already running our commands as root.
+    tty=`echo "${opts}" | jq .tty`
```

Comment:
The dependency on jq is annoying, but it's so useful.

Reply:
https://stackoverflow.com/questions/3919755/how-to-parse-query-string-from-a-bash-cgi-script is a cute alternative. If we only expect simple arguments, we could do that and let people split on `&` in other languages (which is pretty easy).

Reply:
Yeah, agreed. I think it's worth revisiting the SSHFS example once we start adding more bash-friendly output formats so that the jq dependency can be removed. I can add a TODO in the example to reflect that.